pkg/kubectl/cmd/util/clientcache.go

```diff
@@ -53,20 +53,18 @@ type ClientCache struct {
 
 	matchVersion bool
 
-	defaultConfigLock sync.Mutex
-	defaultConfig     *restclient.Config
+	mutex         sync.Mutex
```

Comment:
sorry, didn't catch this before: by convention we use `lock` for the name of the primary lock.